src/g3log/sinkhandle.hpp

```diff
@@ -24,6 +24,7 @@ namespace g3 {
    // in the resulting future. Ref: SinkHandle::call
    template<class T>
    class SinkHandle {
+      friend class LogWorker;
```

Comment:
This is required because SinkHandle::sink() method is protected and it is accessed from LogWorker::removeSink(...)